packages/ts/react-grid/src/autogrid.tsx

```diff
@@ -136,10 +139,14 @@ function useColumns(
       </ColumnContext.Provider>
     );
   });
+  let columns = autoColumns;
   if (options.customColumns) {
-    return [...autoColumns, ...options.customColumns];
+    columns = [...columns, ...options.customColumns];
   }
-  return autoColumns;
+  if (options.rowNumbers) {
+    columns = [<GridColumn width="4em" renderer={AutoGridRowNumberRenderer}></GridColumn>, ...columns];
```

Comment:
The column should define a `key`

Reply:
They are wrapped into the column context, which has the key:
https://github.com/vaadin/hilla/blob/3f284b057a1eacaebe87c03a5dd496c8ec997905/packages/ts/react-grid/src/autogrid.tsx#L131-L136

Reply:
The delete column in auto crud has the same issue BTW:
https://github.com/vaadin/hilla/blob/3f284b057a1eacaebe87c03a5dd496c8ec997905/packages/ts/react-grid/src/autocrud.tsx#L26-L29

Reply:
Why do the children of a ColumnContext not require a key?

Reply:
Ah, it has only one child. Never mind